src/pages/examples/rule.mdx

```diff
@@ -7,7 +7,8 @@ import Rule from '../../components/examples/Rule';
 
 # Rule
 
-This example uses a [Rule](/docs/uischema/rules) that controls the enablement of a control if the 'Is Alive' checkbox is unchecked.
+This example uses two [rules](/docs/uischema/rules). The first rule controls the enablement of a control if the 'Is Alive' checkbox is unchecked.
```

Comment:
Needs to be updated because of the example restructuring
```suggestion
This example uses two [rules](/docs/uischema/rules). The first rule manages the enablement of a control based on whether the 'Is Dead' checkbox is checked.
```